modules/tinymce/src/core/main/ts/keyboard/EnterKey.ts

```diff
@@ -25,6 +27,13 @@ const handleEnterKeyEvent = (editor: Editor, event: EditorEvent<KeyboardEvent>)
   });
 };
 
+const manageEnterInFigure = (editor: Editor) => {
```

Comment:
Could we rename this to `manageEnterOnNonEditable` and could we only run this if the selected node is non-editable by checking the `NodeType.isContentEditableFalse` also should we prevent the default event if we do handle this so should the each maybe be an exists or something. Then have a if statement and a `e.preventDefault()` at the call side.

Reply:
refactored:
- name change
- check on CEF added
- default prevented